source/_integrations/probe_plus.markdown

```diff
@@ -0,0 +1,70 @@
+---
+title: Probe Plus
+description: Instructions on how to integrate Probe Plus food temperature probes into Home Assistant.
+ha_release: 2025.5.0
+ha_category:
+  - Sensor
+ha_iot_class: local_push
+ha_config_flow: true
+ha_domain: probe_plus
+ha_platforms:
+  - sensor
+ha_bluetooth: true
+ha_codeowners:
+  - '@pantherale0'
+ha_integration_type: device
+ha_quality_scale: bronze
+---
+
+The **Probe Plus** {% term integration %} allows you to view food temperature probe details supported by [Probe Plus](https://play.google.com/store/apps/details?id=com.yscoco.thermoex) through Home Assistant.
+
+If your probe is within Bluetooth range to your Home Assistant host and the [Bluetooth](/integrations/bluetooth) integration is fully loaded, the probe should be discovered automatically provided that the model name starts with FM2. If you are configuring the device manually, your probe needs to be turned on during setup. 
+
+Once the integration is set up, Home Assistant will try to connect to your probe every 15 seconds. This means there is sometimes a small delay between you removing the probe from its dock and Home Assistant connecting to it.
+
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+Device:
+  description: "The Bluetooth device that is your probe."
+{% endconfiguration_basic %}
+
+## Available platforms & entities
+
+### Sensors
+
+- **Battery**: Current battery level of the relay and probe.
+- **Temperature**: Current temperature reading from the probe.
+- **Probe Signal Strength**: The signal strength of the probe to its charging dock.
```

Comment:
```suggestion
- **Probe signal strength**: The signal strength of the probe to its charging dock.
```
we use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/capitalization)